README.md

```diff
@@ -37,6 +37,8 @@ They can still be built using CMake directly.
 | Example                                               | Libraries                | Description                                       | GPUs           |
 |:------------------------------------------------------|:-------------------------|:--------------------------------------------------|:---------------|
 | mfem/hypre/magnetic-diffusion.cpp                     | MFEM+HYPRE               | Steady state magnetic diffusion problem           | ![cuda] ![hip] |
+| dealii/petsc_trilinos/petsc_trilinos.cpp              | deal.II+PETSc/Trilinos   | Poisson problem using MPI and AMG preconditioners |                |
+| dealii/sundials/sundials.cpp                          | deal.II+Sundials         | Nonlinear, minimal surface problem                |                |
```

Comment:
These new examples actually work with xSDK v0.8.0, so they should be moved in the table above.